src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -344,6 +344,30 @@ public function setAnnotatedClassCache(array $annotatedClasses)
     }
 
     /**
+     * Return a list of classes to be cached away when the container is build.
+     *
+     * If a class contains annotations, add it to getAnnotatedClassesToCompile()instead.
+     *
+     * @return string[]
+     */
+    public function getClassesToCompile()
```

Comment:
@Ener-Getick if not, we could always patch it to only create and load this in php 5